tests/unit/data/test_data_utils.py

```diff
@@ -156,4 +155,16 @@ def test_scaffold(molecule_dataset_with_rings):
         datapoints=molecule_dataset_with_rings, sizes=(0.3, 0.3, 0.3), split=split_type
     )
 
-    assert train == [0, 1, 2]
+    assert train[0] == [0, 1, 2]
+
+
+def test_splits_from_file(mol_data, data_dir):
+    """
+    Testing if splits_from_file yields expected results.
+    """
+    splits_file = data_dir / "example_splits_file.toml"
+    train, val, test = splits_from_file(datapointss=[mol_data], splits_file=splits_file)
+
+    expected_result = [[[mol_data[idx] for idx in [0, 1, 2, 4]]]]
```

Comment:
This would be better and I could use pytest's tmp_path fixture as the save location for the file. But I decided to just test parse_indices instead, so no file reading writing is needed now.